src/system/ThemeModule/EventListener/Core3UpgradeListener.php

```diff
@@ -35,7 +35,7 @@ public function upgrade(GenericEvent $event): void
             return;
         }
         if ($this->columnExists('themes', 'xhtml')) {
-            $sql = 'ALTER TABLE `themes` DROP COLUMN `xhtml`';
+            $sql = 'ALTER TABLE `themes` DROP COLUMN `xhtml`;';
```

Comment:
please remove this. it is unrelated to this PR and the theme table will be dropped by me anyway.

Reply:
done